test/downloadutils.js

```diff
@@ -43,7 +43,7 @@ function downloadFile(file, url, callback, redirects) {
   protocol.get(url, function (response) {
     var redirectTo;
     if (response.statusCode === 301 || response.statusCode === 302 ||
-        response.statusCode === 307 || response.statusCode === 308) {
+      response.statusCode === 307 || response.statusCode === 308) {
```

Comment:
Revert all whitespace changes in this patch. We try to enforce as much as possible with `eslint`, but these are unfortunately not caught. The code style guide is at https://github.com/mozilla/pdf.js/wiki/Style-Guide.